.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest a feature or improvement
+title: "[FEATURE]: "
+labels: kind/enhancement
+assignees: ''
+
+---
+
+<!-- MAKE SURE TO APPLY SUITABLE LABELS BEFORE FILING THE ISSUE, ESPECIALLY PRIORITY -->
+<!-- AND `kind/documentation` IF IT INVOLVES DOCUMENTATION IN ANY WAY e.g. README -->
```

Comment:
This somehow implies that documentation does not have bugs and any changes to it are feature requests, which might not be the case.

Reply:
```suggestion
```